generation_config.yaml

```diff
@@ -0,0 +1,60 @@
+gapic_generator_version: 2.40.1-SNAPSHOT
+protoc_version: '25.3'
+googleapis_commitish: 5a0ae652df1ef39b7849bf2bd6ba417376ce3204
+libraries_bom_version: 26.38.0
+owlbot_cli_image: sha256:623647ee79ac605858d09e60c1382a716c125fb776f69301b72de1cd35d49409
```

Comment:
I think we can remove these two parameters as they are part of the generation image now? @diegomarquezp

Reply:
Done.